.github/workflows/universalnrp-test-run.yml

```diff
@@ -115,6 +125,8 @@ jobs:
           EOL
 
           sudo LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/opt/omi/lib/ pwsh -Command $script
+          ls -l
+          sudo chmod a+r *testResults.xml
```

Comment:
Thought that sudo was failing on copying the NRP log, not the test report XML. Where's that one?